### ISSUE TITLE
Wrap eCalc component in `webviz-subsurface-components` component

### ISSUE THREAD

**Issue by anders-kiaer** [equinor/webviz-subsurface-components#509]

**Comment by anders-kiaer** [comment]
Need a meeting with eCalc wrt. input conversion/source.

**Comment by TeeeJay** [comment]
hi Anders, is this issue related to using the JSON that we now output from eCalc in the CLI with the `--flow-diagram` option. Is it still blocked, or just put on hold (got question internally about it)

**Comment by anders-kiaer** [comment]
@TeeeJay Indeed, it was originally blocked by the missing availability of `json` data from eCalc CLI. Now it is blocked by the "eCalc + Drogon" example setup task (which I think is close to be finished...).
